pkg/utils/utils.go

```diff
@@ -302,6 +302,23 @@ func getNetdevMTU(pciAddr string) int {
 	return mtu
 }
 
+func getNetMac(pciAddr string) string {
```

Comment:
can we use netlink to get the effective mac?

Reply:
changed